trojsten/rules/test.py

```diff
@@ -1146,11 +1163,11 @@ def test_puzzle_hunt_participations(self):
         generator = SUSIResultsGenerator(self.tag)
         self.assertEqual(
             generator.get_user_coefficient(self.test_user, self.round),
-            susi_constants.SUSI_EXP_POINTS_FOR_PUZZLEHUNT,
+            1 * susi_constants.SUSI_EXP_POINTS_FOR_PUZZLEHUNT,
         )
 
     def test_coefficient_all(self):
-        # Coefficient = 14: successful semesters = 1, other camps = 2,
+        # Coefficient = 15: successful semesters = 2, other camps = 2,
```

Comment:
Máme niekde test, kde sú aspoň dva semestre s 8 vyriešenými úlohami? Ak nie, bolo by dobré taký spraviť/upraviť tento. Toto vraví successful semesters = 2, ale 1 je za sústredko a druhý za úlohy, takže túto situáciu netestuje.

Reply:
pridal som do tohoto testu nech su dva semestre s 8 ulohami